modules/combined/tests/phase_field_fracture/crack2d.i

```diff
@@ -48,18 +48,20 @@
     G0_var = 'G0_pos'
     dG0_dstrain_var = 'dG0_pos_dstrain'
   [../]
+  [./DynamicTensorMechanics]
```

Comment:
OK, that's fine.  I just wanted to make sure it was intentional.  Someone might pick this up and get confused.